src/main/java/seedu/address/logic/commands/NotesCommand.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Notes;
+import seedu.address.model.person.Person;
+
+/**
+ * Changes or adds the notes of an existing person in BizBook.
+ */
+public class NotesCommand extends Command {
+
+    public static final String COMMAND_WORD = "notes";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Edits the notes of the person identified "
+            + "by the index number used in the last person listing. "
+            + "New note will be appended to the notes currently stored.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "n/ [NOTES]\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "n/ High profile client.";
+
+    public static final String MESSAGE_ARGUMENTS = "Index: %1$d, Notes: %2$s";
+
+    public static final String MESSAGE_ADD_NOTES_SUCCESS = "Added notes to Person: %1$s";
+    public static final String MESSAGE_DELETE_NOTES_SUCCESS = "Removed notes from Person: %1$s";
```

Comment:
Should delete note be in the same command, or would it be better to rename this to AddNotesCommand, and implement a DeleteNotesCommand in future.

Reply:
I can make it in the same command and have a different keyword trigger delete, "notes 1 delete" for example. Do you think is okay or is separating into another command class better? 
I didn't separate as the rest of the command classes didn't do this either.

Reply:
I think delete should be another command, reserving the word "delete" could result in unintended behaviors. Example: what if the user wants to create a note with the word "delete"? While this may not make send in a realistic usage, we have to consider that testers will be trying to look for these kind of situation. Thus, I feel that it is better for us to be explicit in its behavior.